CONTRIBUTING.md

```diff
@@ -73,6 +73,17 @@ VisiData has two main branches:
 
 All pull requests should be submitted against `develop`. Submitters will need to e-sign a Copyright Assignment Agreement (CAA) before a pull request will be accepted.
 
+# Documentation and Help Text
+
+Follow these guidelines when writing documentation for VisiData, such as the help text for menu items:
+
+1. Use International English
```

Comment:
What's a good link/reference for learning how to use good International English?